src/Libraries/Microsoft.Extensions.AI.AzureAIInference/AzureAIInferenceChatClient.cs

```diff
@@ -96,6 +96,7 @@ public async Task<ChatResponse> GetResponseAsync(
         // Create the return message.
         ChatMessage message = new(ToChatRole(response.Role), response.Content)
         {
+            MessageId = response.Id, // There is no per-message ID, but there's only one message per response, so use the response ID
```

Comment:
I _think_ this is the right answer. The other would be to generate a unique ID for each message. Did you consider and dismiss that? Just want to make sure we've thought it through.

Reply:
I did consider returning a new GUID or something but don't see who it would help. If we can return an ID that actually came from the service then at least it will mean something if people are searching through logs or telemetry data to coordinate the ID with real-world activity.